ol-keycloak/ol-spi/src/main/java/authentication/authenticators/browser/UsernamePasswordForm.java

```diff
@@ -27,11 +25,6 @@ public void authenticate(AuthenticationFlowContext context) {
             LoginFormsProvider form = context.form();
             realmIdentityProvidersList.removeAll(identityProvidersLinkedWithUser);
             String requestURI = session.getContext().getUri().getBaseUri().getPath();
-            try {
```

Comment:
Appears that the `getBaseUri` method returns an object and that's why the try-catch block is failing.